src/range.rs

```diff
@@ -10,7 +13,7 @@ pub enum RangeItem {
 }
 
 /// Range of possible values, comprised of individual values and/or intervals.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)] //Serialize might be risky due to large memory allocations with Soapy
```

Comment:
Can you explain what this comment means?

Reply:
I noticed that these large memory allocations have nothing to do with the serialization, but it seems to be a bladeRF issue. When calling `dev.frequency_range(Rx, 0).unwrap()` somehow a large amount of memory is allocated until the RAM is full and the application crashes. This does not happen with an RTL-SDR stick. Thus i would remove this comment again.